### ISSUE TITLE
It's too easy to deconstruct tables.

### ISSUE THREAD

**Issue by Uristthedorf** [unitystation/unitystation#3629]
It's entirely possible for one man, with a wrench, to deconstruct all tables on the station within 5 minutes.

Maybe make it a multi step process that takes longer?

**Comment by Evan-R-Rogers** [comment]
On top of this, and slightly unrelated, tables are supposed to deconstruct into their respective 'table parts' instead of just the base wood or metal paneling.

**Comment by ReiderJack** [comment]
https://tgstation13.org/wiki/Guide_to_construction

**Comment by Evan-R-Rogers** [comment]
@ReiderJack 

Odd how theres no mention of table parts and yet in the project there are some hidden away in files.

**Comment by Uristthedorf** [comment]
> https://tgstation13.org/wiki/Guide_to_construction

Ok, I guess you make tables in important areas reinforced, but keep other tables the the way they are now.

**Comment by daalta** [comment]
Agreed, integrities need to be revisisted and compared to how they're supposed to work on TG
